src/components/NcInputField/NcInputField.vue

```diff
@@ -441,7 +441,7 @@ export default {
 		margin-inline: var(--input-padding-start) var(--input-padding-end);
 		max-width: fit-content;
 		font-size: var(--input-label-font-size);
-		inset-block-start: calc((var(--default-clickable-area) - var(--default-line-height)) / 2); // center the label vertically
+		inset-block-start: calc((var(--default-clickable-area) - 1lh) / 2); // center the label vertically
```

Comment:
This is not `Automated styling update from server` 👀

Reply:
I can move it to separate PR, but it'll take more efforts

Reply:
and dark.css isn't updated for some reason..

Reply:
> and dark.css isn't updated for some reason..

Yes, it is not automated. Same with Talk Desktop — default styles are available from source, but theming required a PHP server